pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -337,32 +364,38 @@ def test_to_datetime_tz_psycopg2(self):
         assert is_datetime64_ns_dtype(i)
 
         # tz coerceion
-        result = pd.to_datetime(i, errors='coerce')
+        result = pd.to_datetime(i, errors='coerce', cache=cache)
         tm.assert_index_equal(result, i)
 
-        result = pd.to_datetime(i, errors='coerce', utc=True)
+        result = pd.to_datetime(i, errors='coerce', utc=True, cache=cache)
         expected = pd.DatetimeIndex(['2000-01-01 13:00:00'],
                                     dtype='datetime64[ns, UTC]')
         tm.assert_index_equal(result, expected)
 
-    def test_datetime_bool(self):
+    bool_skip = pytest.mark.skipif(True, reason="GH 18111")
+
```

Comment:
just write this skip inline (don’t use a temp)
you can format so that it fits by putting things on separate lines